pkg/util/util.go

```diff
@@ -121,15 +121,15 @@ func GetAvailableSpace(path string) (int64, error) {
 
 // GetAvailableSpaceBlock gets the amount of available space at the block device path specified.
 func GetAvailableSpaceBlock(deviceName string) (int64, error) {
-	// Check if device exists.
-	info, err := os.Stat(deviceName)
+	// Check if the file exists and is a device file.
+	info, err := os.Lstat(deviceName)
 	if os.IsNotExist(err) {
 		return int64(-1), nil
 	}
-	if info.IsDir() {
+	if (info.Mode() & os.ModeDevice) == 0 {
```

Comment:
A small helper `isDevice(info.Mode()) ` would make it more readable

Reply:
@nijinashok wdyt?

Reply:
Ack, thanks for the review. Added `isDevice` helper.